Makefile

```diff
@@ -63,6 +65,10 @@ install:
 	--set ggircs.namespace=$(GGIRCS_PROJECT) \
 	--set ggircs.prefix=$(GGIRCS_NAMESPACE_PREFIX) \
 	--set ggircs.environment=$(ENVIRONMENT) \
+	--set download-airflow-ciip-cert-renewal.dagConfiguration="{\"org\": \"bcgov\", \"repo\": \"cas-ciip-portal\", \"ref\": \"$(GIT_SHA1)\", \"path\": \"dags/ggircs_cert_renewal.py\"}" \
+	--set download-airflow-ciip-deploy-db.dagConfiguration="{\"org\": \"bcgov\", \"repo\": \"cas-ciip-portal\", \"ref\": \"$(GIT_SHA1)\", \"path\": \"dags/ciip_deploy_db.py\"}" \
+	--set airflow-ciip-deploy-db.helm.hook="pre-upgrade" \
+	--set airflow-ciip-deploy-db.helm.hookWeight=0 \
```

Comment:
those could be set in the values file, as they are always the same value. The "download" dependency should have a `pre-upgrade,pre-install` hook, but the actual DAG trigger does not need to be a hook